docs/installation/server-config/index.rst

```diff
@@ -29,58 +31,58 @@ For example:
 .. code:: bash
 
     bootstrap.servers=localhost:9092
-    listeners=http://localhost:8080
+    listeners=http://localhost:8088
     ui.enabled=true
 
-After you have configured your properties file, you can start KSQL with your properties file specified.
+After you have updated the server configuration file, you can start the KSQL server with the configuration file
+specified.
 
 .. code:: bash
 
     $ <path-to-confluent>/bin/ksql-server-start <path-to-confluent>/etc/ksql/ksql-server.properties
 
-.. tip:: The command topic name is determined by the :ref:`ksql-service-id` configuration.
-
-For more information, see :ref:`ksql-parm-reference`.
+For more information, see :ref:`ksql-param-reference`.
 
 .. _configuring-ksql:
 
+-----------
 JMX Metrics
 -----------
 
 .. include:: ../../includes/ksql-includes.rst
```

Comment:
This is unrelated to this PR, but I noticed this while double-checking:  @joel-hamill it looks to me as if the latest `./docs/includes/ksql-includes.rst` file has JMX information at different start/end lines.  I think the correct numbers are now:

```
.. include:: ../../includes/ksql-includes.rst
:start-line: 325
:end-line: 332
```

Could you please confirm and follow-up with a separate PR if needed?

Reply:
And probably we need to double check the start/end lines for other includes, too.

Reply:
very good point. reviewing all now.

Reply:
@miguno fixed https://github.com/confluentinc/ksql/pull/1112